gcp/rakefiles/entrypoint.rake

```diff
@@ -121,8 +121,7 @@ end
 desc "Undeploy GPII compoments and destroy cluster"
 task :destroy => [:set_vars, @gcp_creds_file, @serviceaccount_key_file, @kubectl_creds_file, :set_secrets] do
   # Terraform will fail if template files are missing
-  Rake::Task[:deploy_module].invoke('k8s/templater')
-  sh "#{@exekube_cmd} down"
+  sh "#{@exekube_cmd} sh -c 'xk up live/#{@env}/k8s/templater && xk down'"
```

Comment:
I don't understand this change. The old version seems functionally equivalent, but takes advantage of the abstraction provided by `:deploy_module`.

Reply:
Since `values_dir` is not mounted from host machine anymore, all templates vanish after docker-compose container terminated. It makes no sense now to invoke templater separately from main exekube command.

Reply:
Ok. Can you add a comment explaining this?

Reply:
Done